### DISCUSSION TITLE
How to change font?

### DISCUSSION THREAD

**Discussion by AndrewILWilliams** [alshedivat/al-folio#215] [Q&A] [answered]
Hi there! I'm just trying out this great project and was wondering how do you change the default font? 

Thanks!
Andrew

**Comment by AndrewILWilliams** [comment]
Oh sorry, I didn't know about the discussions feature! Thanks

**Comment by alshedivat** [comment, marked as answer]
No problem! Re: your question about the font -- to change the fonts, you would need to specify them in one of the style files (e.g., in [`_sass/_base.scss`](https://github.com/alshedivat/al-folio/blob/master/_sass/_base.scss)). By default, all fonts are inherited from [Material Bootstrap](https://mdbootstrap.com/) and changed only sparsely for some of the elements.

**Reply by AndrewILWilliams** [reply]
Thanks! I'm very new to CSS and this type of web design, would you be able to give a basic example? I'm not really sure how to specify the different fonts, or where to find a list of the fonts? Thanks again!

**Reply by AndrewILWilliams** [reply]
Actually, don't worry I think I found it! Thanks though :)

**Reply by muralikgs** [reply]
Hey, I'm trying to change the fonts as well! Can you share any resources that you may have found? It would really helpful!

**Reply by rohandebsarkar** [reply]
See #265

**Reply by gubazoltan** [reply]
I think my question is related to this discussion: 
How to increase the font-size of the al-folio team globally? 

I want to increase all the text (and basically everything) on the al-folio template. Like zooming in on the website but without actual zooming. Thanks in advance!

**Reply by george-gca** [reply]
See #1515

**Comment by pengzhenghao** [comment]
Sorry just came across a similar problem. I saw many places we are using `rem` as the font-size unit. Do we have any global variable to control what 1 rem equals to what?

I've tried changing the body's font-size, and it seems not effecting the rem related objects.

**Reply by george-gca** [reply]
Check [this post](https://www.sitepoint.com/understanding-and-using-rem-units-in-css/) for information.
